src/high-level/include/iDynTree/KinDynComputations.h

```diff
@@ -697,6 +697,39 @@ class KinDynComputations {
      */
     bool generalizedGravityForces(FreeFloatingGeneralizedTorques & generalizedGravityForces);
 
+    /**
+     * Compute the getNrOfDOFS()+6 vector of generalized external forces.
+     *
+     * The semantics of baseAcc, the base part of baseForceAndJointTorques
```

Comment:
IMHO using "variable name style", i.e. `baseAcc` and `baseForceAndJointTorques` in a function that does not accept these variables is a bit confusing.
Same for the `inverseDynamicsInertialParametersRegressor` method.

Reply:
Yes, that's a left over, thanks for catching that.